nixos/modules/services/web-apps/bookstack.nix

```diff
@@ -132,12 +136,15 @@ in
       "database"
       "user"
     ] "Use services.bookstack.settings.DB_USERNAME instead.")
-    (lib.mkRemovedOptionModule [
-      "services"
-      "bookstack"
-      "database"
-      "createLocally"
-    ] "This option is deprecated. Please create your database manually.")
+    (lib.mkRemovedOptionModule
+      [
+        "services"
+        "bookstack"
+        "database"
+        "createLocally"
+      ]
+      "This option is deprecated. Please create your database manually or use services.mysql.ensureDatabases and services.mysql.ensureUsers."
```

Comment:
The first sentence must be dropped because a removed option cannot be used at all while a deprecated can and mkRemovedOptionModule already prefixes the message with that. 

Also advising people to do things manually is not very NixOS. We should recommend the options as they are doing what is required and only in exceptions advise to do things manually.